src/legacy/core_plugins/visualizations/public/embeddable/get_index_pattern.ts

```diff
@@ -29,8 +28,8 @@ export async function getIndexPattern(
     return savedVis.vis.indexPattern;
   }
 
-  const savedObjectsClient = npStart.core.savedObjects.client;
-  const defaultIndex = npStart.core.uiSettings.get('defaultIndex');
+  const savedObjectsClient = getSavedObjectsClient().client;
```

Comment:
Either `getSavedObjectsClient` should return the client, or its name should be `getSavedObjects`